### ISSUE TITLE
Escape special Markdown symbols

### ISSUE THREAD

**Issue by Fechin** [donatj/CsvToMarkdownTable#9]
**Input csv**：
```
id,name
1,Name|sok
```
**Output Markdown**：
This is a wrong markdown table because it has not been escaped
```
| id | name     | 
|----|----------| 
| 1  | Name|sok | 
```

Can you do the same as [tableconvert.com](https://tableconvert.com/) to support special character escaping?

**Comment by Fechin** [comment]
**Correct result**：
```
| id | name      |
|----|-----------|
| 1  | Name\|sok |
```

**Comment by donatj** [comment]
This was fixed with the now published v1.0.0 release.

https://github.com/donatj/CsvToMarkdownTable/releases/tag/v1.0.0

Thanks for the report.
